lib/shared-sku-app/src/SkuPicker/styles.ts

```diff
@@ -1,5 +1,5 @@
-import { css } from 'emotion';
-import tokens from '@contentful/forma-36-tokens';
+import { css } from "emotion";
```

Comment:
shouldn't these be single quotes according to the prettier rule?

Reply:
yup, run it for the file but prettier is messed up all over the apps repo when the code was migrated over unfortunately :/